package.json

```diff
@@ -7,39 +7,44 @@
   "author": "jbcpollak <josh@6river.com> (http://6river.com/)",
   "main": "server/server.js",
   "engines": {
-    "node": "5.7.1",
+    "node": "6.2.1",
     "npm": "3.7.2"
   },
   "scripts": {
-    "debug": "NODE_ENV=development node debug .",
-    "start": "NODE_ENV=development nodemon .",
+    "debug": "NODE_ENV=development NODE_PATH=$NODE_PATH:./ node debug .",
+    "start": "NODE_ENV=development NODE_PATH=$NODE_PATH:./ nodemon .",
     "pretest": "eslint .",
-    "test": "NODE_ENV=test istanbul cover node_modules/.bin/_mocha",
-    "test:no-cover": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test NODE_PATH=$NODE_PATH:./ istanbul cover node_modules/.bin/_mocha -- ${MOCHA_OPTS}",
+    "test:no-cover": "NODE_ENV=test NODE_PATH=$NODE_PATH:./ mocha",
```

Comment:
same, also add ${MOCHA_OPTS} to the end of this line, and we should be specing the `test` directory too:

```
    "test": "NODE_ENV=test istanbul cover node_modules/.bin/_mocha -- ${MOCHA_OPTS} test",
    "test:no-cover": "NODE_ENV=test mocha test",
```